app/models/comfy/cms/page.rb

```diff
@@ -47,6 +48,10 @@ def self.in_category(category_id)
       .reorder('comfy_cms_categorizations.ordinal ASC')
   end
 
+  def translation(locale = :cy)
```

Comment:
Yeah, you alright. I was following the same concept about PageMirror, but is a different context. I will change that.